tests/xlsx/xlsx_export.test.ts

```diff
@@ -679,6 +679,29 @@ describe("Test XLSX export", () => {
         "Conditional formats with formula rules are not supported at the moment. The rule is therefore skipped."
       );
     });
+
+    test("Conditional formatting with DataBar Rule is correctly exported", async () => {
+      const model = new Model({
+        sheets: [
+          {
+            colNumber: 2,
+            rowNumber: 5,
+            cells: {},
+            conditionalFormats: [
+              {
+                id: "1",
+                ranges: ["B1:B5"],
+                rule: {
+                  type: "DataBarRule",
+                  color: 0xffffff,
```

Comment:
nitpick: probably a bit more readable to use `hexaToInt(#ffff)` same in the import test